jenkins/scripts/ci-node-version.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -ex
+
+rm -rf src node_version.h.tar || true
+
+source env.properties
+
+set +x
+export GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i $JENKINS_TMP_KEY"
+set -x
+
+# Retry once
+git archive --format=tar --remote="$TEMP_REPO" $TEMP_BRANCH src/node_version.h -o node_version.h.tar ||
+git archive --format=tar --remote="$TEMP_REPO" $TEMP_BRANCH src/node_version.h -o node_version.h.tar
```

Comment:
this is interesting, fetching a single file from a remote repo? I get protocol problems trying this out locally so can't seem to replicate this behaviour.

Reply:
I also had issues with http repos, but this works for our temp repo (ssh). This is already being used by node-test-commit-windows-fanned (running in jenkins-workspace).